lib/membership/member.js

```diff
@@ -91,10 +90,14 @@ Member.prototype.evaluateUpdate = function evaluateUpdate(update) {
     if (this._isLocalOverride(update)) {
         // Override intended update. Assert aliveness!
         this.ringpop.stat('increment', 'refuted-update');
-        update = _.defaults({
+        var newIncNumber = this.Date.now();
+        update = {
+            source: this.ringpop.whoami(),
+            sourceIncarnationNumber: newIncNumber,
+            address: this.id,
             status: Member.Status.alive,
-            incarnationNumber: this.Date.now()
-        }, update);
+            incarnationNumber: newIncNumber
+        };
```

Comment:
I guess we can look at it both ways (forwards compatibility versus risk of not changing fields that should change), and I don't have a strong preference here. So feel free to do either way.